src/components/InventoryGroups/Modals/CreateGroupModal.test.js

```diff
@@ -1,25 +1,144 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
 import { validateGroupName } from '../utils/api';
-import { validate } from './CreateGroupModal';
+import CreateGroupModal, { validate } from './CreateGroupModal';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 
 jest.mock('../utils/api');
+jest.mock('react-redux');
```

Comment:
For my understanding: Why the whole react-redux is mocked?

Reply:
This helps to mock useDispatch call in the modal component. Yep, this mock is very general, it can be enhanced in the future if we would need different behaviour for different function from the library. For now, as it serves its purpose for this test, we can leave it like that :)